### ISSUE TITLE
ORCID login issue

### ISSUE THREAD

**Issue by maurerle** [openjournals/joss#1401]
Whenever I click the `Submit` Button, which leads to https://joss.theoj.org/papers/new - I am signed out of ORCID.

Clicking "login using ORCID" again, directly passes as the OAuth token is still valid, but it is still very annoying.
I don't know if anyone else experiences this, but I did not find a related issue so far.

**Comment by kyleniemeyer** [comment]
I also find that I get signed out frequently, particularly if I try to work in multiple JOSS website-related windows. I haven't been able to resolve it myself, but I agree it's quite annoying.

**Comment by maurerle** [comment]
Yes, I think this is related to the problem - when I open a link in a new tab, I am not signed in in this tab anymore..
Clicking the login button directly logs in without password as the OIDC/OAuth session of the authentication provider at ORCID is still valid..

Maybe this helps.

Browser:
Firefox 128.5.0esr

**Comment by arfon** [comment]
I think this might be the root cause: https://github.com/openjournals/joss/issues/1428 (fix in the works).

**Comment by warrickball** [comment]
I also used to find myself being logged out quite frequently but haven't had trouble for many months now. I just tried logging in to JOSS with ORCID in a clean browser session then navigating to `papers/new`, and I wasn't logged out. I'm going to pre-emptively close this but we can re-open it if necessary.
